src/byteweight/options.ml

```diff
@@ -0,0 +1,35 @@
+open Core_kernel.Std
```

Comment:
I think last time we made a deal that we would keep byteweight in its own directory until we move symbols and options out as part of the library.